apis/coursebatchmanapi/coursebatchmanapi.yaml

```diff
@@ -56,7 +56,843 @@ paths:
+             - $ref: '#/definitions/BatchCreateErrorResponse'
+  /batch/read/{Batch_ID}:
+    parameters:
+      - in: path
+        name: Batch_ID
+        type: string
+        required: true
+        description: 'Append a valid Batch ID To the Request URL'   
+    get:
+      description: >-
+        This API is associated with fetching a particular batch on the Sunbird Platform.
+          - {...}refers to a section of read/{Batch_Id} endpoint URL path that needs to be replaced by appropriate Id
+          - The **_/batch/read/{Batch_Id}_** endpoint executes "Fetch a batch" request to fetch a particular batch.
+          - It is a must to provide values for required parameters 
+      summary: Fetch a particular Batch
+      tags:
+      - Course Batch APIs
+      operationId: getBatch
+      produces:
+      - application/json
+      parameters:
+      - name: Content-Type
+        in: header
+        required: true
+        type: string
+        description:  >-
+            The Content Type entity is the media type of the resource.Possible media types can be:- 
+              - Application/json
+      - name: ts
+        in: header
+        required: true
+        type: string
+        description: 'The time stamp at which the **Fetch a batch** request was sent. '
+      - name: X-msgid
+        in: header
+        required: true
+        type: string
+        description: 'It is a Unique ID which identifies the request incase the same API is executed multiple times.'
+      - name:  x-authenticated-user-token
+        in: header
+        required: true
+        type: string
+        description: 'The alphanumeric string for accessing the API'
+      - name: Authorization
+        in: header
+        required: true
+        type: string
+        description: To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received, here.'
+      responses:
+        '200':
+          description: 'SUCCESS. The **Fetch a batch** operation was successful!'
+          schema:
+            allOf:
+             - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+             - $ref: '#/definitions/SingleBatchResponseWrapper'
+            
+        '400':
+          description:  'BAD REQUEST,**Fetch a batch** operation failed !The possible reason for failure is that you may have missed providing input for a mandatory parameter'
+          schema:
+            allOf:
+             - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+             - $ref: '#/definitions/BatchCreateErrorResponse'
+        '404':
+          description: 'RESOURCE NOT FOUND" operation failed ! The possible reason for failure is that BatchId is invalid.'

```

Comment:
RESOURCE NOT FOUND, ** _____________** operation failed!
Possible failure reason: Entered an invalid Course ID

Please provide the name of the failed operation